analytics.json

```diff
@@ -0,0 +1,7 @@
+[
+  "appnexus",
+  "ga",
+  "pulsepoint",
+  "roxot",
+  "sharethrough_analytics"
```

Comment:
let's remove these as this will change the default behavior of building with all the analytics adapters.